src/plugins/embeddable/public/lib/panel/embeddable_panel.tsx

```diff
@@ -187,6 +189,11 @@ export class EmbeddablePanel extends React.Component<Props, State> {
     if (this.embeddableRoot.current) {
       this.props.embeddable.render(this.embeddableRoot.current);
     }
+    this.props.embeddable.actionStorage.count().then(drilldownCount => {
```

Comment:
Please fetch the latest changes, it should be now `embeddable.dynamicActions`, hopefully the final interface.